tickets/src/components/content/CreateContent.js

```diff
@@ -149,23 +175,35 @@ export class CreateContent extends Component {
 
 CreateContent.propTypes = {
   now: PropTypes.instanceOf(Date).isRequired,
+  account: UnlockPropTypes.account,
+  addEvent: PropTypes.func.isRequired,
   locks: PropTypes.arrayOf(PropTypes.string),
 }
 
 CreateContent.defaultProps = {
   locks: [],
+  account: null,
 }
 
-export const mapStateToProps = ({ locks }) => {
+export const mapStateToProps = ({ locks, account }) => {
   let selectLocks = []
   Object.values(locks).map(lock => selectLocks.push(lock.address))
+  let now = new Date()
   return {
     locks: selectLocks,
-    now: new Date(),
+    account,
```

Comment:
nit: rather than passing the account down, maybe handle this in `mapDispatchToProps`?